flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorTest.java

```diff
@@ -198,14 +198,6 @@ public void broadcastEvent(AbstractEvent event, boolean isPriorityEvent)
                             throws IOException {
                         super.broadcastEvent(event, isPriorityEvent);
                         broadcastedPriorityEvent.set(isPriorityEvent);
-                        // test if we can write output data
```

Comment:
FORCE_ALIGNED checkpoint has no channel state on input side. But the barrier is translated into its prior form in `SubtaskCheckpointCoordinator` and thus can have output data written.
This portion actually tests that the conversion succeeded and output data can be written.

Reply:
You're right. And the checkpoint **is** actually started, just IDs don't match (`0` here and `42` below).
I'll revert this change correct the checkpoint id.

Reply:
Oh shoot, how did I miss that?